pkg/config/checks/hpaMaxAvailability.yaml

```diff
@@ -0,0 +1,4 @@
+successMessage: HPA has a valid max and min replica configuration
+failureMessage: HPA maxReplicas should be greater than minReplicas
+category: Reliability
+target: autoscaling/HorizontalPodAutoscaler
```

Comment:
Since no schema is provided, it will use the function validator defined by the same checkID (`hpaMaxAvailability`)